config/betterquesting/DefaultQuests.json

```diff
@@ -112792,12 +112792,6 @@
               "Count:3": 1,
               "Damage:2": 0,
               "OreDict:8": ""
-            },
```

Comment:
Wasn't this necessary for crafting or why this is removed?

Reply:
The diamond sword is not necessary for crafting, it was just a requirement for the quest based on "needing a good sword" but by this point a player can have many options for a sword. See #5233.